index.html

```diff
@@ -273,6 +275,32 @@
                 Security and Privacy Considerations</a> repository to file issues, 
                as they are cross-cutting over all our specifications.
         </p>
+
+    <p>This document describes an abstract architecture design.
+    However, there is a
+    <a href="https://w3c.github.io/wot-thing-description/testing/report.html">
+    preliminary implementation report</a> which describes concrete
+    implementations based on the associated <em>WoT Thing Description</em>
+    specification. The Working Group seeks implementation feedback,
+    having set the requirement of at least two implementations of each
+    feature as the exit criteria for the Candidate Recommendation
+    phase. The group aims to obtain reports from one TD producer and
+    one TD consumer for each feature if applicable.  For details,
+    including definitions of implementation, TD producer, and TD
+    consumer, see the
+    <a href="https://w3c.github.io/wot-thing-description/testing/report.html">
+    preliminary implementation report</a>.
+    </p>
+
+    <p>This document was published by the <a href=
+    "https://www.w3.org/WoT/WG/">Web of Things Working Group</a> as
+      an Editor's Draft.
+     This document is intended to become a <abbr title="World Wide Web Consortium">W3C</abbr> Recommendation.</p>
+
+    <p>Please see the Working Group's
+    <a href="https://w3c.github.io/vc-test-suite/implementations/">
```

Comment:
this should be 'https://w3c.github.io/wot-thing-description/testing/report.html'.

Reply:
fixed.

Reply:
fixed.